Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3
+
+RUN pip install --no-cache-dir minitor
+WORKDIR /app
+COPY config.yml /app/config.yml
```

Comment:
`config.yml` is not in this base repo. You can use `sample-config.yml` and then others could add a volume that replaces it